### PR TITLE
Add hooks

### PR DESCRIPTION
**Pull request by huseyinbiyik**
You can find the features added for this Pull Request below.

-Add React hooks
-Change the component from class-based to function

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,51 +1,81 @@
...
-        <button className="zero" type="button" onClick={this.onClickHandler}>0</button>
-        <button className="point" type="button" onClick={this.onClickHandler}>.</button>
-        <button className="equal" type="button" onClick={this.onClickHandler}>=</button>
+  const { total, operation, next } = state;
+  return (
+    <div className="wrapper">
+      <div className="total">
+        {total}
+        {operation}
+        {next}
```
Comment (DeliceLydia) [inline src/components/Calculator.js]: - [ ] Please avoid using the `function` keyword instead use ES6 `callback` functions to make your code more modern and more readable. It allows you to write less code and do more, you can simply do:
```
const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);
```

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @huseyinbiyik, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- App is working well
- Awesome README
- No linter errors


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: src/components/Calculator.js (hunk 1 above)
Comment (huseyinbiyik) [reply src/components/Calculator.js]: Oops. I forgot to convert the main function. Thank you for your review!

**Review by julie-ify — APPROVED** [review]
Hi @huseyinbiyik, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**